.github/actions/test/action.yml

```diff
@@ -1,5 +1,17 @@
 name: "Test"
 description: "Run end-to-end tests. Note that this action assumes that Node and Next.js have already been properly setup. See the setup-next task for details."
+inputs:
+  api-key:
+    description: Replay.io API Key
+    # This API key allows access to the whole team that holds all these test
+    # recordings. If someone wanted to, they could go in and delete the workspace
+    # or recordings in it or anything. While thats not great, it's also not
+    # the end of the world. If someone does that, we can always change
+    # this code to only run in CI in the main repo and have this as a secret.
+    # It's a lot easier to hardcode it for now though.
+    # trunk-ignore(gitleaks/generic-api-key)
+    default: rwk_yaEG8jo6gcisGHHoMj8SNoOMIHSbT7REuU5E1QnKCiL
```

Comment:
Excited for this to go away someday! What's the long-term plan here?

Reply:
Team inboxes, i think